pkg/components/external-dns/component.go

```diff
@@ -118,27 +117,27 @@ func (c *component) RenderManifests() (map[string]string, error) {
 	if c.AwsConfig.AccessKeyID == "" {
 		accessKeyID, ok := os.LookupEnv("AWS_ACCESS_KEY_ID")
 		if !ok || accessKeyID == "" {
-			return nil, errors.New("AWS Credentials not found.")
+			return nil, fmt.Errorf("AWS Access Key ID not found")
```

Comment:
Is it valid to use `fmt.Errorf()` when no string formatting is used?

Reply:
I didn't find any linter or rule being against it. Otherwise we would have to still import 3rd part `errors` for `errors.New`. This way, we have consistent convention using core library.